examples/CalcQt/CMakeLists.txt

```diff
@@ -1,21 +1,49 @@
 project(CalcQt)
 
-find_package(Qt4 COMPONENTS QtCore QtGui QtTest)
+set(CALCQT_HEADERS src/CalculatorWidget.h)
+set(CALCQT_SOURCES src/CalcQt.cpp src/CalculatorWidget.cpp)
+set(LIBCALCQT_BUILD OFF)
```

Comment:
Is `LIBCALCQT_BUILD` really needed? If it's for the `if(LIBCALCQT_BUILD)` later on, can't you use the `QT_LIBRARIES` variable?